src/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  outline: none;
+  height: 100%;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 300;
```

Comment:
just for the future, don't use `*` to reset all elements' styles. mostly we use `*` only for `box-sizing: border-box` and `html` for common font styles.  other styles move to some special tags, classes. it's just FYI, you can leave it as it's for this task.